moveit_planners/ompl/ompl_interface/src/ompl_planner_manager.cpp

```diff
@@ -58,6 +60,51 @@ class OMPLPlannerManager : public planning_interface::PlannerManager
 public:
   OMPLPlannerManager() : planning_interface::PlannerManager(), nh_("~"), display_random_valid_states_(false)
   {
+      class OutputHandler : public ompl::msg::OutputHandler
+      {
+      public:
+          OutputHandler() : ompl::msg::OutputHandler()
+          {
+          }
+
+          ~OutputHandler() override
+          {
+          }
+
+          void log(const std::string &text, ompl::msg::LogLevel level, const char *filename, int line) override
+          {
+              std::unique_lock<std::mutex> lock(lock_);
```

Comment:
Do you really need locking? ROS_CONSOLE / the underlying logging uses locking as well.